arroyo/processing/processor.py

```diff
@@ -425,9 +425,7 @@ def _run_once(self) -> None:
             if self.__message is not None:
                 try:
                     start_submit = time.time()
-                    message = (
```

Comment:
IDK why this was here before, let me know if removing this is a mistake

Reply:
This is different now. If `self.__message` is None now it would return `Message(None)`. Before it would have returned `None`. Is this change intentional ?

Reply:
i think this change is correct since the entire block only runs if message is not none (see line 425) -- probably a copypaste error or to pacify mypy

Reply:
yeah that block does not even run